src/test/java/net/sf/jabref/logic/importer/fileformat/BibtexParserTest.java

```diff
@@ -935,39 +935,39 @@ public void parseRecognizesPreamble() throws IOException {
 
         ParserResult result = BibtexParser.parse(new StringReader("@preamble{some text and \\latex}"),
                 importFormatPreferences);
-        assertEquals("some text and \\latex", result.getDatabase().getPreamble());
+        assertEquals(Optional.of("some text and \\latex"), result.getDatabase().getPreamble());
     }
 
     @Test
     public void parseRecognizesUppercasePreamble() throws IOException {
 
         ParserResult result = BibtexParser.parse(new StringReader("@PREAMBLE{some text and \\latex}"),
                 importFormatPreferences);
-        assertEquals("some text and \\latex", result.getDatabase().getPreamble());
+        assertEquals(Optional.of("some text and \\latex"), result.getDatabase().getPreamble());
     }
 
     @Test
     public void parseRecognizesPreambleWithWhitespace() throws IOException {
 
         ParserResult result = BibtexParser.parse(new StringReader("@preamble {some text and \\latex}"),
                 importFormatPreferences);
-        assertEquals("some text and \\latex", result.getDatabase().getPreamble());
+        assertEquals(Optional.of("some text and \\latex"), result.getDatabase().getPreamble());
     }
 
     @Test
     public void parseRecognizesPreambleInParenthesis() throws IOException {
 
         ParserResult result = BibtexParser.parse(new StringReader("@preamble(some text and \\latex)"),
                 importFormatPreferences);
-        assertEquals("some text and \\latex", result.getDatabase().getPreamble());
+        assertEquals(Optional.of("some text and \\latex"), result.getDatabase().getPreamble());
     }
 
     @Test
     public void parseRecognizesPreambleWithConcatenation() throws IOException {
 
         ParserResult result = BibtexParser.parse(new StringReader("@preamble{\"some text\" # \"and \\latex\"}"),
```

Comment:
What happens for, e.g., `new StringReader("@preamble{}` and `new StringReader(""}`